tests/grid/filtered_iterator.cc

```diff
@@ -140,9 +140,9 @@ void test ()
       bool (*predicate) (const active_cell_iterator, const unsigned int)
         = &level_equal_to<active_cell_iterator>;
       FilteredIterator<active_cell_iterator>
-      begin (std_cxx11::bind (std::ptr_fun(predicate), std_cxx11::_1, 3),
+      begin (std_cxx11::bind (predicate, std_cxx11::_1, 3),
```

Comment:
Yes, I suspect that at one point the `FilteredIterator` class simply took a general template argument and was later converted to `std::function<...>`, for which we then wrapped everything into `std::bind`.

Either way, nice simplification!